pkg/db/db.go

```diff
@@ -2120,19 +2145,25 @@ func (h *DBHandler) RunCustomMigrationsCommitEvents(ctx context.Context, getAllE
 				}
 			}
 		}
+		//Migration event
+		err = h.writeEvent(ctx, transaction, 0, "00000000-0000-0000-0000-000000000001", event.EventTypeDBMigrationEventType, strings.Repeat("0", 40), []byte("{}"))
```

Comment:
why the "1" at the end?
```suggestion
		err = h.writeEvent(ctx, transaction, 0, "00000000-0000-0000-0000-000000000000", event.EventTypeDBMigrationEventType, strings.Repeat("0", 40), []byte("{}"))
```

Reply:
Please extract the special UUID into a constant, or a function that returns it.